### DISCUSSION TITLE
Business 20250208 Practice room supplier proposal

### DISCUSSION THREAD

**Discussion by Cheng-ChiAn** [J-tour/J-tour.github.io#29] [General]
# 新竹市練團室
### 鴻韻樂器
https://www.hymusic.com.tw/tympanum/
![image](https://github.com/user-attachments/assets/5e57e083-5f9a-4132-8cf1-81e5cc6d9f9b)
>練團室租借價位：350元／HR，出示學生證300元／HR
地址：新竹市復興路16-5號
電話：03-5266133-11
營業時間
週一:17:00-22:00
週二:13:00-22:00
週三:10:00-22:00
週四:13:00-22:00
週五:13:00-22:00
週六:09:00-22:00
週日及國定假日 公休
### 灰街樂器-大同店
https://graystreetmusic.com/%e5%a0%b4%e5%9c%b0%e7%a7%9f%e5%80%9f/
![image](https://github.com/user-attachments/assets/94933ac8-7f35-495c-b58d-a755d1dedf48)
>練團室租借價位：300元／HR，FB打卡優惠 250/小時。
地址：新竹市大同路154號
電話： 03-5258705
營業時間
週一至週五:12:00-22:00
週六:10:00-22:00
週日:10:00-20:00
# 中壢市練團室
### 俠客樂器-中壢店
https://www.sharkmusic.com.tw/h/ServiceDetail?key=e3n79&set=7&cont=415472
![image](https://github.com/user-attachments/assets/61cb83f4-2caf-4559-ac7f-44285f5ac46f)
>練團室租借價位：350元／HR
地址：中壢市中台路33號〔近中正公園〕
電話： 03-427-0083
營業時間
週一至週五:13:00-22:00
週六週日:10:00-22:00
### 焦點音樂Focus Music
https://www.facebook.com/p/%E7%84%A6%E9%BB%9E%E9%9F%B3%E6%A8%82-Focus-Music-100063737238260/
![image](https://github.com/user-attachments/assets/3d8db5b4-3c7c-4b78-bcc5-08e8fb944a59)
>練團室租借價位：390元／HR
地址：桃園市中壢區中壢區復興路71號12樓
電話： 03 425 4250
營業時間
週一:下午4:00 - 下午10:00
週二:未營業
週三:下午4:00 - 下午10:00
週四:下午4:00 - 下午10:00
週五:下午4:00 - 下午10:00
週六:下午12:00 - 下午10:00
週日:下午12:00 - 下午10:00
未完 待續

**Comment by Pighome7** [comment]
我偏好鴻韻樂器、俠客樂器這兩家用來拍照

鴻韻樂器的燈光和背景比較好看 
俠客樂器有沙發椅適合用來拍照

**Reply by Cheng-ChiAn** [reply]
收到

**Reply by ShengHaoLo** [reply]
@Cheng-ChiAn 請山峰拍形象照那天是否需要進練團室棚內？
